### ISSUE TITLE
False positive "Unlabeled break inside of labeled block"

### ISSUE THREAD

**Issue by Undin** [intellij-rust/intellij-rust#4111]
## Environment

* **Intellij-Rust plugin version:** 0.2.101.2678-192-nightly
* **Rust toolchain version:** rustc 1.38.0-nightly (dfd52ba6a 2019-07-06)
* **IDE name and version:** IU-192.5587.17
* **Operating system:** macOS 10.13.6

## Problem description
![image](https://user-images.githubusercontent.com/2539310/60775109-c821d380-a126-11e9-93ad-20caecee52b7.png)

## Steps to reproduce
```rust
#![feature(label_break_value)]

fn main() {
    let b = 'b: {
        let a = |x: i32| break;
        1
    };
}
```
